packages/flutter/test/gestures/scale_test.dart

```diff
@@ -1,7 +1,7 @@
 // Copyright 2015 The Chromium Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
-
+import 'dart:math' as math;
```

Comment:
please keep the blank line above the imports. also, add a blank line between the dart: and package: imports.